include/kllvm/ast/AST.h

```diff
@@ -102,11 +104,19 @@ class KORESymbol {
   virtual void print(std::ostream &Out, unsigned indent = 0) const =0;
 };
 
+struct HashSymbol;
+
+class KOREDefinition;
+class KOREObjectSymbolDeclaration;
+
 class KOREObjectSymbol : public KORESymbol {
 private:
   std::string name;
```

Comment:
Please add comments for member variables. What to firstTag and lastTag do? What is layout?